packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -938,6 +938,25 @@ const defaultConfigs = {
     uniswapAddress: "0x360acfeb5c1548bad3583c559a646d803077236d",
     twapLength: 7200,
     invertPrice: false
+  },
+  USDXIO: {
```

Comment:
Alternative configuration that follows the specs in the UMIP. I've tested and this looks to return the expected prices, but please test on your end!

```
  USDXIO: {
    type: "expression",
    expression: `
      USDETH * ETHXIO
    `,
    minTimeBetweenUpdates: 60,
    lookback: 7200,
    customFeeds: {
      ETHXIO: {
        type: "uniswap",
        uniswapAddress: "0xe0cc5afc0ff2c76183416fb8d1a29f6799fb2cdf",
        invertPrice: true,
        twapLength: 3600
      }
    }
  }
```

Reply:
Thank you @smb2796, your suggested config is working fine. I have created another commit.